raiden/tasks.py

```diff
@@ -110,11 +110,11 @@ def check_rdn_deposits(
             click.secho(
                 (
                     f"WARNING\n"
-                    f"Your account's RDN balance of {rdn_balance} is below the "
-                    f"minimum threshold. Provided that you have either a monitoring "
-                    f"service or a path finding service activated, your node is not going "
-                    f"to be able to pay those services which may lead to denial of service or "
-                    f"loss of funds."
+                    f"Your account's RDN balance deposited in the UserDepositContract"
+                    f"of {rdn_balance} is below the minimum threshold."
```

Comment:
shouldn't it also have the minimum threshold here?

Reply:
good point, makes total sense